packages/dev-middleware/src/__tests__/FetchUtils.js

```diff
@@ -21,7 +21,7 @@ declare var globalThis: $FlowFixMe;
  */
 export async function fetchLocal(
   url: string,
-  options?: Parameters<typeof fetch>[1] & {dispatcher?: mixed},
```

Comment:
I had to add this, without this it required me to add _all_ request options as second parameter, while I just want to do:
```ts
const response = await fetchLocal('..', {method: 'POST'})
```